site2/docs/reference-configuration.md

```diff
@@ -142,6 +142,7 @@ Pulsar brokers are responsible for handling incoming messages from producers, di
 |statsUpdateFrequencyInSecs||60|
 |statsUpdateInitialDelayInSecs||60|
 |metadataStoreUrl| Metadata store quorum connection string  ||
+| metadataStoreConfigPath | The configuration file path of the local metadata store, see [here](administration-metadata-store.md) for details. |N/A|
```

Comment:
```suggestion
| metadataStoreConfigPath | The configuration file path of the local metadata store. See [Configure metadata store](administration-metadata-store.md) for details. |N/A|
```
Use alternative text to "here" for better accessibility. Other occurrences need an update as well.